packages/edit-post/src/components/block-manager/index.js

```diff
@@ -38,6 +41,20 @@ function BlockManager( {
 				includes( blockType.parent, 'core/post-content' ) )
 	);
 
+	// Announce search results on change
+	useEffect( () => {
+		if ( ! search ) {
+			return;
+		}
+		const count = blockTypes.length;
+		const resultsFoundMessage = sprintf(
+			/* translators: %d: number of results. */
+			_n( '%d result found.', '%d results found.', count ),
+			count
+		);
+		debouncedSpeak( resultsFoundMessage );
+	}, [ blockTypes, search, debouncedSpeak ] );
```

Comment:
I noticed that other implementations were omitting filtered values from dependencies. Is there a reason for that, or it's just a missing deps case?

Reply:
It seems like it was just missed.. Also maybe we could add as dependency just the length of `blockTypes`? 🤔